core/src/main/java/org/elasticsearch/search/aggregations/pipeline/PipelineAggregatorBuilders.java

```diff
@@ -99,6 +102,10 @@ public static BucketSelectorPipelineAggregationBuilder bucketSelector(String nam
         return new BucketSelectorPipelineAggregationBuilder(name, script, bucketsPaths);
     }
 
+    public static BucketSortPipelineAggregationBuilder bucketSort(String name, List<FieldSortBuilder> sorts) {
```

Comment:
Why is field sort builder specified, instead of a more flexible sort builder ?

Reply:
Because it can only refer to a bucket path in the parent aggregation.